systemtest/src/test/java/io/strimzi/systemtest/security/oauth/OauthPlainST.java

```diff
@@ -222,6 +224,7 @@ void testProducerConsumerMirrorMaker() {
     @Test
     @Tag(MIRROR_MAKER2)
     @Tag(CONNECT_COMPONENTS)
+    @Tag(NODEPORT_SUPPORTED)
```

Comment:
I wonder why we have nodePort listener here when we don't use it at all...guess we should double-check it and remove/use it in some next PR.

Reply:
There're quite a few instances of tests like this!

Reply:
ye....I will open an issue for it :)